course_catalog/settings.py

```diff
@@ -21,14 +21,14 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = get_string('SECRET_KEY', None)
+SECRET_KEY = get_string('SECRET_KEY', "terribly_unsafe_default_secret_key")
```

Comment:
Keep as is, if `SECRET_KEY` is not set in the `.env` file then the app will not start, which is intended: 
```
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```